langchain/src/vectorstores/weaviate.ts

```diff
@@ -126,6 +130,7 @@ export class WeaviateStore extends VectorStore {
 
       const flattenedMetadata = flattenObjectForWeaviate(document.metadata);
       return {
+        ...this.tenant ? { tenant: this.tenant } : {},
```

Comment:
`...this.tenant` looks like a typo?

Reply:
I think you mean 
```typescript
...(this.tenant ? { tenant: this.tenant } : {}),
```

Reply:
Not a typo but I do agree it's more readable with the parenthesis. I've updated as per your suggestion.